src/lib/y2users/user.rb

```diff
@@ -134,6 +148,13 @@ def initialize(name)
       @system = false
     end
 
+    # Default home for the user
+    #
+    # @return [Home]
```

Comment:
I would document that it creates a new instance, so all previous home related config is lost.